app/views/patients/show.html.erb

```diff
@@ -148,7 +148,7 @@
     </table>
     <%= react_component('subject/LastDateExposure', {
                           authenticity_token: form_authenticity_token,
-                          patient: @patient,
+                          patient: @patient_hash,
```

Comment:
How does this not break https://github.com/SaraAlert/SaraAlert/blob/master/app/javascript/components/subject/LastDateExposure.js#L280?

Reply:
After a rebase this component is not here anymore

Reply:
But I did fix this issue https://github.com/SaraAlert/SaraAlert/pull/592/commits/def66e7e4020f231729dd5ef51afb5128bb05f5a